src/main/java/spoon/reflect/visitor/DefaultJavaPrettyPrinter.java

```diff
@@ -567,7 +567,7 @@ public void visitCtCatch(CtCatch catchBlock) {
 		elementPrinterHelper.writeComment(catchBlock, CommentOffset.BEFORE);
 		printer.writeSpace().writeKeyword("catch").writeSpace().writeSeparator("(");
 		CtCatchVariable<? extends Throwable> parameter = catchBlock.getParameter();
-		if (parameter.getMultiTypes().size() > 1) {
+		if (parameter != null && parameter.getMultiTypes() != null && parameter.getMultiTypes().size() > 1) {
```

Comment:
parameter.getMultiTypes() is never null. I suggest to remove this check. 
WDYT?